src/main/java/com/autotune/analyzer/recommendations/engine/RecommendationEngine.java

```diff
@@ -1294,19 +1305,26 @@ private ValidationOutputData addRecommendationsToDB(Map<String, KruizeObject> ma
         return validationOutputData;
     }
 
-    private void getResults(Map<String, KruizeObject> mainKruizeExperimentMAP, KruizeObject kruizeObject, String
+    private String getResults(Map<String, KruizeObject> mainKruizeExperimentMAP, KruizeObject kruizeObject, String
             experimentName, Timestamp intervalStartTime, String dataSource) {
+        String errorMsg = "";
         // get data from the DB in case of remote monitoring
         if (kruizeObject.getExperiment_usecase_type().isRemote_monitoring()) {
             mainKruizeExperimentMAP.put(experimentName, kruizeObject);
             try {
-                new ExperimentDBService().loadResultsFromDBByName(mainKruizeExperimentMAP, experimentName, intervalStartTime, interval_end_time);
+                boolean resultsAvailable = new ExperimentDBService().loadResultsFromDBByName(mainKruizeExperimentMAP, experimentName, intervalStartTime, interval_end_time);
+                if (!resultsAvailable) {
+                    errorMsg = String.format("%s%s", MISSING_INTERVAL_END_TIME, intervalEndTimeStr);
```

Comment:
Would it be acceptable to remove this error message and instead convey "No metrics or results were found from/to this date"?

Reply:
Sounds good. Updated now